Header header
int16 cel
int16 cer
int16 cef
int8 cgl
int8 cgr
int8 cgf
